actions/train/zergling.py

```diff
@@ -21,9 +21,15 @@ def __init__(self, ai):
         if not self.ai.can_train(ZERGLING):
             return False
 
-        if self.ai.units(ULTRALISKCAVERN).ready and self.ai.time < 1380:
+        if self.ai.caverns.ready and self.ai.time < 1380:
             if not len(self.ai.ultralisks) * 8.5 > len(self.ai.zerglings):
                 return False
+
+        if self.ai.spires.ready:
+            if self.ai.supply_used > 150:
+                return False
+            if not len(self.ai.mutalisks) * 10 > len(self.ai.zerglings):
```

Comment:
why do we bind zerglings to muta amount if we have a spire?

Reply:
Tweakimp said it prioritizes zerglings over mutas, so I bound a minimum amount of mutas to be created, 10 zerglings to 1 muta, until supply used reaches 150

Reply:
its the same thing we do with ultras see?
```
        if self.ai.caverns.ready and self.ai.time < 1380:
            if not len(self.ai.ultralisks) * 8.5 > len(self.ai.zerglings):
                return False
```

Reply:
the problem was that we create 200 supply of zergling and then can not kill floating buildings because we can not build mutas.